File: arrays_and_strings/1-2.js
// Given two strings, write a method to determine if one is a permuatation of the other

function determinePermutation(str1, str2) {
  return str1.split("").sort().join("") === str2.split("").sort().join("")
}

console.log(determinePermutation("breeew", "what's good"))
console.log(determinePermutation("ay", "ya"))
console.log(determinePermutation("whatdoe", "doewhat"))

File: arrays_and_strings/1-2.rb
# Given two strings, write a method to determine if one is a permuatation of the other

def permutation?(str1, str2)
  return str1.split("").sort == str2.split("").sort
end

p permutation?("oh hey", "what's good?")
p permutation?("sup brew", "bup srew")